config/routes.php

```diff
@@ -22,13 +22,13 @@
     $map->addRoute(['GET', 'POST'], '/album[/]', 'modules/album/index.php');                         // Photo Album
     $map->addRoute(['GET', 'POST'], '/downloads[/]', 'modules/downloads/index.php');                 // Downloads
     $map->addRoute(['GET', 'POST'], '/forum[/]', 'modules/forum/index.php');                         // Forum
-    $map->addRoute(['GET', 'POST'], '/guestbook[/]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
+    $map->addRoute(['GET', 'POST'], '/guestbook[/[{action}[/[{id:\d+}[/]]]]]', 'modules/guestbook/index.php');                 // Guestbook, mini-chat
```

Comment:
Please don't make unnecessary changes to one commit. If commit is dedicated to translating comments into English, that's all it should be. Changing Route makes a problem to Guestbook.

Reply:
I will pay attention to it in the next time

Reply:
I have consulted some people to make this decision. The change also makes the router simpler and more concise later 😆